test/e2e/node/node_problem_detector.go

```diff
@@ -239,7 +239,7 @@ func verifyNodeCondition(f *framework.Framework, condition v1.NodeConditionType,
 	return nil
 }
 
-func getMemoryStat(f *framework.Framework, host string) (rss, workingSet float64) {
+func getMemoryStat(host string) (rss, workingSet float64) {
```

Comment:
Just before this, there's function where it is being called.
	node, err := f.ClientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})

Reply:
> ode, err := f.ClientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})

this particular code snippet  is in function [verifyNodeCondition], 
not  in the function of  [getCPUStat] or [getMemoryStat] ,

so, Will it also be affected if the parameter “(f *framework.Framework) in function [getCPUStat] or [getMemoryStat] ” is deleted directly?